docs/sphinx/using/backends/hardware/neutralatom.rst

```diff
@@ -130,6 +130,85 @@ Submitting
 
 
 
+Pasqal
+++++++++++++++++
+
+Pasqal is a quantum computing hardware company that builds quantum processors from ordered neutral atoms in 2D and 3D
+arrays to bring a practical quantum advantage to its customers and address real-world problems.
+The currently available Pasqal QPUs are analog quantum computers, and one, named Fresnel, is available through our cloud
+portal.
+
+In order to access Pasqal's devices you need an account for `Pasqal's cloud platform <https://portal.pasqal.cloud>`__
+and an active project. Although a different interface, `Pasqal's Pulser library <https://pulser.readthedocs.io/en/latest/>`__, is a good
+resource for getting started with analog neutral atom quantum computing. For support you can also use `Pasqal Community <https://community.pasqal.com/>`__.
+
+
+.. _pasqal-backend:
+
+Setting Credentials
+```````````````````
+
+An authentication token for the session must be obtained from Pasqal's cloud platform.
```

Comment:
Is there a link that will always have up to date instructions on how to get a token?

Reply:
Our documentation portal https://docs.pasqal.cloud/. I will add it - I see that it's missing